Alluvial.Distributors.Sql/SqlBrokeredDistributor.cs

```diff
@@ -84,11 +81,11 @@ public class SqlBrokeredDistributor<T> : DistributorBase<T>
             {
                 cmd.CommandType = CommandType.StoredProcedure;
                 cmd.CommandText = @"Alluvial.AcquireLease";
-                cmd.Parameters.AddWithValue(@"@waitIntervalMilliseconds", WaitInterval.TotalMilliseconds);
+                cmd.Parameters.AddWithValue(@"@waitIntervalMilliseconds", 0);
```

Comment:
For the moment, yes. There isn't currently a migration strategy for the sprocs so I'd prefer to keep the test coverage using the existing sproc definition.